src/components/signupform/SignupForm.js

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react'
+import CustomizableButton from '../common/CustomizableButton'
+import styles from './signupform.module.css'
+
+function Form() {
+  const [user, setUser] = useState({
+    name: '',
+    email: '',
+    password: '',
+  })
+  const handleChange = (e) => {
+    setUser({ ...user, [e.target.name]: e.target.value })
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    if (validateForm()) {
+      console.log('Form submitted sucessfully')
+      // Will direct the user to the Login Form
+      setTimeout(() => {
+        resetForm()
+      }, 1000)
+    }
+  }
+
+  const resetForm = () => {
+    setUser({ name: '', email: '', password: '' })
+  }
+
+  const validateForm = () => {
+    if (!user.name || !user.email || !user.password) {
+      alert('please fill out all required fields.')
```

Comment:
An alert is a good start. Instead of using alert() to display validation errors in the next iteration of this component should add an error message element under each input field and display true on this element when validation is failed. Then have the text appear in red.

This kind of validation exists typically out of the box with frontend frameworks like Bootstrap. If you plan to add one of these frameworks then ignore my suggestion.

Reply:
> An alert is a good start. Instead of using alert() to display validation errors in the next iteration of this component should add an error message element under each input field and display true on this element when validation is failed. Then have the text appear in red.
> 
> This kind of validation exists typically out of the box with frontend frameworks like Bootstrap. If you plan to add one of these frameworks then ignore my suggestion.

Yes,  I had planned to add something else later beyond the alert box. Just need to discuss if we're allowed to bring in a framework like Bootstrap.

Reply:
Yea, I think we should use a framework because it comes built-in with all of these little touches. I feel its something we should have started with though because it would have been easier but no issues then 😄

Reply:
@TaylorDJones11 let's raise using Bootstrap in Sprint Planning